### PR TITLE
Update definitions for react-toggle

### PR DESCRIPTION
**Pull request by LKay**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-toggle/react-toggle-tests.tsx [hunk 1]
```diff
@@ -1,10 +1,36 @@
 import * as React from "react";
-import Toggle from "react-toggle";
+import Toggle, { ToggleIcons } from "react-toggle";
 
 class Test extends React.Component {
+    handleEvent = (e: any) => {};
```
Comment (Kovensky) [inline types/react-toggle/react-toggle-tests.tsx]: not sure if typing this as `any` makes it much of a test...

`e: React.SyntheticEvent<any>`?

File: types/react-toggle/index.d.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 export interface ToggleIcons {
     checked?: ReactNode;
     unchecked?: ReactNode;
 }
 
-export interface ToggleProps extends HTMLAttributes<any> {
+export interface ToggleProps extends InputHTMLAttributes<HTMLInputElement> {
     "aria-labelledby"?: string;
     "aria-label"?: string;
```
Comment (Kovensky) [inline types/react-toggle/index.d.ts]: It think `aria-` props are special-cased by the parser to always be allowed as string props...... but I'm not sure on this. I know that `data-` is.

Comment (RyanCavanaugh) [inline types/react-toggle/index.d.ts]: Names like `aria-` and `data-` are always allowed as type `any`, except if there's a declared property with that name (in which case it's enforced as usual). So this is a good addition

**Review by Jessidhia — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
@Kovensky thanks and sorry for the bot spam

**Pull request merged**